docs/source/tools/ledger-api-test-tool/index.rst

```diff
@@ -0,0 +1,127 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-api-test-tool:
+
+Ledger API Test Tool
+####################
+
+The Ledger API Test Tool is a command line tool for testing the correctness of
+ledger implementations based on DAML and the :doc:`Ledger API
+</app-dev/ledger-api-introduction/index>`. It’s useful for ledger implementation
+developers, who are using the DAML Ledger Implementation Kit to develop a DAML
+Ledger on top of their distributed-ledger or database of choice.
+
+Use this tool to verify if the Ledger API endpoint conforms to the :doc:`DA
+Ledger Model </concepts/ledger-model/index>`.
+
+.. contents:: On this page:
+  :local:
+
+Downloading the tool
+====================
+
+TODO
+
+Extracting :code:`.dar` file required to run the tests
+======================================================
+
+Before you can run the Ledger API test tool on your ledger, you need to load a
+specific set of DAML templates onto your ledger.
+
+#. Obtain the corresponding :code:`.dar` file with:
+
+   ``ledger-api-test-tool -x``
+
+#. Load the file :code:`SemanticTests.dar` created in the current directory into your
+   Ledger.
+
+Running the tool against a custom Ledger API endpoint
+=====================================================
+
+Run this command to test your Ledger API endpoint exposed at host :code:`<host>` and
+at a port :code:`<port>`:
+
+    ``ledger-api-test-tool -h <host> -p <port>``
+
+For example
+
+    ``ledger-api-test-tool -h localhost -p 6865``
+
+Exploring options the tool provides
+===================================
+
+Run the tool with :code:`--help` flag to obtain the list of options the tool provides:
+
+   .. code-block:: console
+
+     $ ledger-api-test-tool --help
+
+     The Ledger API Test Tool is a command line tool for testing the correctness of
+     ledger implementations based on DAML and Ledger API.
+     Usage: ledger-api-test-tool [options]
+
+       --help                   prints this usage text
+       -p, --target-port <value>
+                                Ledger API server port. Defaults to 6865.
```

Comment:
Obviously this is a comment for the output itself, but this should really be "Server port of the Ledger API endpoint to test" or something